Mathlib/Topology/Algebra/InfiniteSum/MultipliableTendstoUniformly.lean

```diff
@@ -0,0 +1,135 @@
+/-
+Copyright (c) 2025 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import Mathlib.Analysis.SpecialFunctions.Log.Summable
+
+/-!
+# Uniform limits of products of functions
+
+We gather some results about the uniform convergence of the products, in particular those of the
+form `1 + f n x` for a sequence `f n x` of complex numbers.
+
+-/
+
+open Filter Function Complex Real
+
+open scoped Interval Topology BigOperators Nat Complex
+
+variable {α β ι : Type*}
+
+@[to_additive] --what do I call this?
+lemma TendstoUniformlyOn_Eventually_lt_mul_const [UniformSpace β] [Group β]
+    [UniformGroup β] [LinearOrder β] [OrderTopology β] [MulLeftStrictMono β] [MulRightMono β]
+    (f : ι → α → β)  {p : Filter ι} {g : α → β} {K : Set α} {T V : β} (htv : T < V)
+    (hf : TendstoUniformlyOn f g p K) (hg : ∀ x, x ∈ K → g x ≤ T) :
+    ∀ᶠ (n : ι) in p, ∀ x, x ∈ K → f n x < V := by
+  rw [@tendstoUniformlyOn_iff_tendsto] at hf
+  simp only [uniformity_eq_comap_inv_mul_nhds_one, tendsto_iff_eventually, eventually_comap,
+    Prod.forall, uniformContinuous_def, mem_comap, forall_exists_index, and_imp] at *
+  rw [Subtype.forall'] at hf
+  have hf2 := hf ⟨(fun x : β × β => x.1⁻¹ * x.2 < T⁻¹ * V), by
+    rw [eventually_iff_exists_mem]
+    refine ⟨{ x : β | x < T⁻¹ * V }, IsOpen.mem_nhds (isOpen_gt' (T⁻¹ * V)) (by simp [htv]) ,
+      fun y hy a b hab => ?_⟩
+    simpa only [Subtype.forall, lt_inv_mul_iff_mul_lt, Set.mem_setOf_eq, hab] using hy⟩
```

Comment:
Here we have a ⟨ ⟩ pattern nested inside another, can it be restructured to avoid that? Also the `simpa only` is terminal so it doesn't need to be squeezed, which gets you a few golf points.